### PR TITLE
docs(tutorial): update tutorial strapi

### PR DESCRIPTION
**Pull request by bene2k1**
### Description

Update tutorial strapi with latest release

### CODE REVIEW

File: tutorials/strapi/index.mdx [hunk 1]
```diff
@@ -32,29 +32,36 @@ In this tutorial you will learn how to deploy Strapi on a [Scaleway Instance](/c
...
+    apt update
+    apt install -y nodejs
+    ```
+6. Check that Node.js and npm are installed:
+    ```
+    node -v && npm -v && npx -v
     ```
 
     You should see an output confirming the installed version number of each.
 6. Strapi uses the yarn package manager. Install the repository for the required version of Yarn by running the following commands:
```
Comment (nerda-codes) [inline tutorials/strapi/index.mdx:61]:
```suggestion
7. Strapi uses the yarn package manager. Install the repository for the required version of Yarn by running the following commands:
```

File: tutorials/strapi/index.mdx [hunk 2]
```diff
@@ -32,29 +32,36 @@ In this tutorial you will learn how to deploy Strapi on a [Scaleway Instance](/c
...
     ```
-    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
-    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
-    apt update && apt install yarn
+    npm install --global yarn
+
     ```
 7. Check that yarn has installed with the following command:
```
Comment (nerda-codes) [inline tutorials/strapi/index.mdx:66]:
```suggestion
8. Check that yarn has installed with the following command:
```

File: tutorials/strapi/index.mdx [hunk 3]
```diff
@@ -73,8 +80,11 @@ In this tutorial you will learn how to deploy Strapi on a [Scaleway Instance](/c
...
     cd strapi-projects
-    yarn create strapi-app my-strapi-project --quickstart
+    npx create-strapi-app@4.14.2 my-strapi-project --quickstart
     ```
+    <Message type="tip">
+      Strapi version 4.14.2 was the latest release available at the time writing this tutorial. Check out their [GitHub repository](https://github.com/strapi/strapi/releases) to find information about the latest Strapi release.
+    </Message>
 
     <Message type="note">
       Using the `--quickstart` flag installs Strapi using an SQLite database. You can omit this flag, but you need to [follow some other steps](https://strapi.io/documentation/3.0.0-beta.x/guides/databases.html#sqlite-installation) to configure an alternative database (Strapi supports [PostgreSQL](/tutorials/install-postgresql/), [MongoDB](/tutorials/mongodb-ubuntu-1604/), SQLite, [MySQL](/tutorials/setup-mysql/), and [MariaDB](/tutorials/mariadb-ubuntu-bionic/)). Your choice of database must be installed and running locally before you create your Strapi project.
```
Comment (nerda-codes) [inline tutorials/strapi/index.mdx:90]:
```suggestion
      Using the `--quickstart` flag installs Strapi using an SQLite database. You can omit this flag, but you need to [follow some other steps](https://strapi.io/documentation/3.0.0-beta.x/guides/databases.html#sqlite-installation) to configure an alternative database (Strapi supports [PostgreSQL](/tutorials/install-postgresql/), [MongoDB](/tutorials/mongodb-ubuntu-1604/), SQLite, [MySQL](/tutorials/setup-mysql/), and [MariaDB](/tutorials/mariadb-ubuntu-bionic/)). Your database of choice must be installed and running locally before you create your Strapi project.
```

File: tutorials/strapi/index.mdx [hunk 4]
```diff
@@ -32,31 +32,38 @@ In this tutorial you will learn how to deploy Strapi on a [Scaleway Instance](/c
...
+7. Strapi uses the yarn package manager. Install the repository for the required version of Yarn by running the following commands:
     ```
-    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
-    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
-    apt update && apt install yarn
+    npm install --global yarn
+
     ```
-7. Check that yarn has installed with the following command:
+8. Check that yarn has installed with the following command:
```
Comment (jcirinosclwy) [inline tutorials/strapi/index.mdx:66]: Here, `yarn` doesn't start with a capital Y?

File: tutorials/strapi/index.mdx [hunk 5]
```diff
@@ -73,11 +80,14 @@ In this tutorial you will learn how to deploy Strapi on a [Scaleway Instance](/c
 2. Install Strapi in this directory and create a new project:
     ```
     cd strapi-projects
-    yarn create strapi-app my-strapi-project --quickstart
+    npx create-strapi-app@4.14.2 my-strapi-project --quickstart
     ```
+    <Message type="tip">
+      Strapi version 4.14.2 was the latest release available at the time writing this tutorial. Check out their [GitHub repository](https://github.com/strapi/strapi/releases) to find information about the latest Strapi release.
```
Comment (jcirinosclwy) [inline tutorials/strapi/index.mdx:86]:
```suggestion
      Strapi version 4.14.2 was the latest release available at the time this tutorial was written. Check out their [GitHub repository](https://github.com/strapi/strapi/releases) to find information about the latest Strapi release.
```

**Review by jcirinosclwy — APPROVED** [review]

**Review by RoRoJ — APPROVED** [review]

File: tutorials/strapi/index.mdx (hunk 4 above)
Comment (bene2k1) [reply tutorials/strapi/index.mdx:66]:
```suggestion
8. Check that `yarn` has installed with the following command:
```

**Pull request merged**